command.go

```diff
@@ -979,7 +979,12 @@ func (c *Command) ExecuteC() (cmd *Command, err error) {
 	return cmd, err
 }
 
+// ValidateArgs returns an error if any positional args are not in
+// the `ValidArgs` field of `Command`
```

Comment:
The comment should probably also include that the `Args` validator is also executed, of specified.

Reply:
Done.